README.md

```diff
@@ -1118,6 +1123,28 @@ Once the job is complete your successful `/status` response will look similar to
 
 The workflow is exactly the same as [creating an image cutout](#example-1-initiate-a-job-to-create-an-image-cutout) except you use the `/mask` endpoint instead of `/cutout`.  
 
+## Customized Workflow
+This section will demonstrate how to make a 'customized workflow' by chaining different APIs. 
+
+### Example 3: (Generate ImageCutOut result as Photoshop path)
+This workflow is ONLY for users who'd like to generate cutout result as Photoshop path instead of regular mask or cutout in above example 1([link](https://github.com/AdobeDocs/photoshop-api-docs#example-1-initiate-a-job-to-create-an-image-cutout)) and example 2([link](https://github.com/AdobeDocs/photoshop-api-docs#example-2-initiate-a-job-to-create-an-image-mask)). You will need to chain API calls to ImageCutOut service and Photoshop Service to achieve this goal. 
```

Comment:
Instead of calling it out as link. Can we make example 2 itself as hyperlinked. Meaning `[example 2]` like this? Same change for the pre-release repo please

Reply:
Ok. Will make the adjustment.

Reply:
Fixed in newest commit